driver-core/src/main/java/com/datastax/driver/core/IdempotenceAwarePreparedStatement.java

```diff
@@ -0,0 +1,42 @@
+/*
+ *      Copyright (C) 2012-2015 DataStax Inc.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+package com.datastax.driver.core;
+
+public interface IdempotenceAwarePreparedStatement extends PreparedStatement {
```

Comment:
Maybe a small doc here could be useful, explaining why we have this new interface (binary compatibility) and that it will be removed in 3.0.